big-practice/src/dashboard.html

```diff
@@ -52,6 +52,31 @@
         </div>
       </section>
     </div>
+    <div class="modal">
+      <div class="modal-content">
+        <div class="modal-table-title">
+          <p class="title table-title">New Movie</p>
+        </div>
+        <form class="form-create-movies">
+          <div class="form-group">
```

Comment:
It should be form-line because you have only 1 input. Form group should include inputs